### PR TITLE
sig-observability: Update ownerships

### PR DESCRIPTION
**Pull request by machadovilaca**
### What this PR does
Before this PR:

We had no assigned observability reviewer, nor approvers/reviewers and sig/observability labeling in observability packages

After this PR:

- @machadovilaca as observability reviewer
- @sradco as observability approver

Owner files in:
- pkg/monitoring
- tests/monitoring
- tests/libmonitoring
- hack/prom-rule-ci

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

/cc @enp0s3 @dhiller @fabiand

### CODE REVIEW

**Review by enp0s3 — COMMENTED** [review]
@machadovilaca Hi, can you please add yourself to these as well:
```
/hack/prom-rule-ci
/tests/monitoring
```

**Comment by machadovilaca** [comment]
> @machadovilaca Hi, can you please add yourself to these as well:
> 
> ```
> /hack/prom-rule-ci
> /tests/monitoring
> ```

added

**Comment by machadovilaca** [comment]
@fabiand updated

**Comment by iholder101** [comment]
Thank you @machadovilaca!

I think it's essential to assign an approver to sig-obvservability as well.
Perhaps @sradco could be a good fit?

**Comment by machadovilaca** [comment]
@enp0s3 @fabiand can you take a new look?

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -126,8 +126,10 @@ aliases:
   # Owns the responsibility to keep kubevirt observable by i.e.
   # having mertics, alters, and runbooks.
   #
-  sig-observability-approvers: []
-  sig-observability-reviewers: []
+  sig-observability-approvers:
+    = sradco
```
Comment (iholder101) [inline OWNERS_ALIASES:130]: typo

```suggestion
    - sradco
```

Comment (machadovilaca) [reply OWNERS_ALIASES]: ups, mb

**Comment by iholder101** [comment]
Looks good to me.
Holding to let @fabiand and others express their opinions and objections, if any.

/lgtm
/hold

**Comment by machadovilaca** [comment]
ping @fabiand

**Comment by fabiand** [comment]
Thanks @machadovilaca please link to a SIG Charter proposal as other SIGs have done in this thread: https://groups.google.com/g/kubevirt-dev/c/9hOLqb9W2-Y

**Comment by fabiand** [comment]
I'm 

/hold

ing until then, but generally

/lgtm

**Comment by machadovilaca** [comment]
sig-observability charter:
mailing list - https://groups.google.com/g/kubevirt-dev/c/9hOLqb9W2-Y
PR - https://github.com/kubevirt/community/pull/282

**Comment by fabiand** [comment]
/unhold

as a charter got posted

**Pull request merged**